app/views/nsm/steps/view_claim/allowed_work_items.html.erb

```diff
@@ -49,45 +49,49 @@
     %>
   <% end %>
 
-  <%
-    pagy, work_items = pagy(@work_items.where.not(adjustment_comment: nil))
-    head = [
-      { text: t('.item'), width: 'govuk-!-width-one-quarter' },
-      { text: t('.time'), numeric: true },
-      { text: t('.uplift'), numeric: true },
-      { text: t('.net_cost'), numeric: true },
-      { text: t('.allowed_time'), numeric: true },
-      { text: t('.allowed_uplift'), numeric: true },
-      { text: t('.allowed_net_cost'), numeric: true },
-      { text: t('.action'), numeric: true }
-    ]
-    work_items.group_by(&:completed_on).each do |completed_on, work_items_for_date| %>
-    <%=
-      rows = work_items_for_date.map do |work_item|
-        next unless work_item.adjustment_comment
+  <% pagy, work_items = pagy(@work_items.where.not(adjustment_comment: nil)) %>
+    <% if work_items.size.zero? %>
+      <p><%= t('.no_data') %></p>
+    <% else %>
+    <%
+      head = [
+        { text: t('.item'), width: 'govuk-!-width-one-quarter' },
+        { text: t('.time'), numeric: true },
+        { text: t('.uplift'), numeric: true },
+        { text: t('.net_cost'), numeric: true },
+        { text: t('.allowed_time'), numeric: true },
+        { text: t('.allowed_uplift'), numeric: true },
+        { text: t('.allowed_net_cost'), numeric: true },
+        { text: t('.action'), numeric: true }
+      ]
+      work_items.group_by(&:completed_on).each do |completed_on, work_items_for_date| %>
+      <%=
+        rows = work_items_for_date.map do |work_item|
+          next unless work_item.adjustment_comment
```

Comment:
This looks redundant - `work_item` comes from `work_items_for_date` which comes from `work_items` which comes from `@work_items.where.not(adjustment_comment: nil)`. I don't think `adjustment_comment` can ever be falsey can it?